packages/core/examples/webLoader/index.ts

```diff
@@ -0,0 +1,110 @@
+import {
+  RenderingEngine,
+  Enums,
+  imageLoader,
+  metaData,
+  getRenderingEngine,
+  Types,
+} from '@cornerstonejs/core';
+import {
+  initDemo,
+  setTitleAndDescription,
+  addSliderToToolbar,
+} from '../../../../utils/demo/helpers';
+import hardcodedMetaDataProvider from './hardcodedMetaDataProvider';
+import registerWebImageLoader from './registerWebImageLoader';
+
+// This is for debugging purposes
+console.warn(
+  'Click on index.ts to open source code for this example --------->'
+);
+
+const { ViewportType } = Enums;
+
+// ======== Set up page ======== //
+setTitleAndDescription(
+  'Web Color Images',
+  'Demonstrates how to render a web color image in JPG or PNG format in a StackViewport'
+);
+
+const content = document.getElementById('content');
+const element = document.createElement('div');
+element.id = 'cornerstone-element';
+element.style.width = '500px';
+element.style.height = '500px';
+
+content.appendChild(element);
+
+const renderingEngineId = 'myRenderingEngine';
+const viewportId = 'COLOR_STACK';
+
+// png images hosted on web (s3 with cors enabled)
```

Comment:
Please indicate that these have no PHI in them and have permission to distribute.